dtfs-central-api/src/cron-scheduler-jobs/delete-acbs-durable-function-logs/index.test.ts

```diff
@@ -10,6 +10,8 @@ jest.mock('../../repositories/durable-functions-repo', () => ({
 }));
 jest.mock('@ukef/dtfs2-common', () => ({
   asString: jest.fn(),
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+  ApiError: jest.requireActual('@ukef/dtfs2-common').ApiError as ApiError,
 }));
```

Comment:
Does the below not work?
```suggestion

jest.mock('@ukef/dtfs2-common', () => ({
  asString: jest.fn(),
  ...jest.requireActual('@ukef/dtfs2-common')
}));
```

And do we need to import `ApiError` on line 1?